### ISSUE TITLE
hyperparameters for .89 accuracy?

### ISSUE THREAD

**Issue by williamFalcon** [jiecaoyu/pytorch-nin-cifar10#5]

**Comment by Kshitij09** [comment]
@williamFalcon I was able to reach .**86** accuracy on Cifar10 with NiN architecture. You could find the details [here](https://github.com/Kshitij09/fast_impl/pull/5)
